bambi/models.py

```diff
@@ -812,20 +812,20 @@ def predict(
         idata : InferenceData
             The ``InferenceData`` instance returned by ``.fit()``.
         kind : str
-            Indicates the type of prediction required. Can be ``"mean"`` or ``"pps"``. The
-            first returns draws from the posterior distribution of the mean, while the latter
-            returns the draws from the posterior predictive distribution (i.e. the posterior
-            probability distribution for a new observation) in addition to the mean posterior
-            distribution. Defaults to ``"mean"``.
+            Indicates the type of prediction required. Can be ``"response_params"`` or
+            ``"response"``. The first returns draws from the posterior distribution of
+            the mean, while the latter returns the draws from the posterior predictive
+            distribution (i.e. the posterior probability distribution for a new observation)
+            in addition to the mean posterior distribution. Defaults to ``"response_params"``.
         data : pandas.DataFrame or None
             An optional data frame with values for the predictors that are used to obtain
             out-of-sample predictions. If omitted, the original dataset is used.
         inplace : bool
             If ``True`` it will modify ``idata`` in-place. Otherwise, it will return a copy of
-            ``idata`` with the predictions added. If ``kind="mean"``, a new variable ending in
-            ``"_mean"`` is added to the ``posterior`` group. If ``kind="pps"``, it appends a
-            ``posterior_predictive`` group to ``idata``. If any of these already exist, it will be
-            overwritten.
+            ``idata`` with the predictions added. If ``kind="response_params"``, a new variable
+            with the name of the parent parameter, e.g. ``"mu"`` or ``"p"``is added to the
```

Comment:
Here it can be multiple parameters (e.g. both mu and sigma)

Reply:
Good point. I updated it to include 2 minimal examples (Gaussian and Bernoulli) so the user can reason what parameters to expect in the posterior group.